### PR TITLE
I1553 projects 3

### PR DESCRIPTION
**Pull request by Serhiy91**
Add DepartmentsSubcard

### CODE REVIEW

File: graphql-server/imports/api/apollo/resolvers/Mutation/Department/deleteDepartment.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  checkLoggedIn,
+  flattenInput,
+  checkDepartmentAccess,
+  deleteAfterware,
+} from '../../../../../share/middleware';
+
+export const resolver = async (root, args, context) =>
+  context.services.DepartmentService.remove(args, context);
+
+export default applyMiddleware(
```
Comment (Dartv) [inline graphql-server/imports/api/apollo/resolvers/Mutation/Department/deleteDepartment.js]: you should check for org settings role

File: graphql-server/imports/api/apollo/resolvers/Mutation/Department/updateDepartment.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  checkDepartmentAccess,
+  departmentUpdateAfterware,
+} from '../../../../../share/middleware';
+
+export const resolver = async (root, args, context) =>
+  context.services.DepartmentService.update(args, context);
+
+export default applyMiddleware(
+  checkLoggedIn(),
+  flattenInput(),
```
Comment (Dartv) [inline graphql-server/imports/api/apollo/resolvers/Mutation/Department/updateDepartment.js]: you should check for org settings role

**Comment by Dartv** [comment]
The easiest solution to stuttering I could think of is setting `keepDirtyOnReinitialize` manually inside of submit callback.
```
setConfig('keepDirtyOnReinitialize', true);
return onCreate(
  .then(() => setConfig('keepDirtyOnReinitialize', false))
  .catch(swal.error);
```

**Comment by Serhiy91** [comment]
@Dartv fixed it here https://github.com/Pliohub/Plio/pull/1568

**Pull request merged**